pynetdicom/transport.py

```diff
@@ -646,81 +475,6 @@ def get_handlers(self, event):
 
         return self._handlers[event]
 
-    def get_request(self):
```

Comment:
note how I removed all the over rides of TCPServer. Instead made it into a protocol factory which can be passed to asyncio.start_server.

Echo server example is here: https://docs.python.org/3/library/asyncio-stream.html#tcp-echo-server-using-streams